### DISCUSSION TITLE
Using the CSV importer with proxy authentication

### DISCUSSION THREAD

**Discussion by jbergler** [firefly-iii/firefly-iii#4502] [Firefly III] [answered]
Hi James - thanks for all the work that goes into this project!

I've deployed with Docker + Authelia. So far so good, the Remote-User header gets parsed.
But that's where I seem to get stuck, it seems that the web app disables anything related to oauth + personal access tokens with this mode enabled, which makes sense - does this mean that this mode is also not compatible with the importing tools that rely on the api?

**Comment by JC5** [comment, marked as answer]
It will be in the new release, but keep in mind you must then exclude the API path from Authelia.

**Comment by shizunge** [comment]
@jbergler Have you tried CSV importer version 2.2.4? (#4533)
Using CSV importer 2.2.4 I could get it authenticated via Personal Access Token without exclude the API path from Authelia.

**Reply by jbergler** [reply]
Sorry, only found time today to have another go at this. 

Yes it works great in the new version.
